test/corpus/recipes.txt

```diff
@@ -551,7 +551,7 @@ recipes containing attributes with arguments
 ================================================================================
 
 
-[group("foo"), group("echo")]
+[group("foo"), group: "echo"]
```

Comment:
the s-expr should not change at all since this is an alternative form of the same statement

Reply:
Could you add this as a new test rather than changing what exists?